api/src/index.ts

```diff
@@ -37,6 +37,8 @@ export {
 export type { DiagAPI } from './api/diag';
 
 // Metrics APIs
+export { ProxyMeter, MeterDelegator } from './metrics/ProxyMeter';
+export { ProxyMeterProvider } from './metrics/ProxyMeterProvider';
```

Comment:
Hmm, would it be required to export this? 🤔 It seems to me that it'd perfectly fine to keep this internal in case we want to change something about it in the future.